tubearchivist/home/src/es/index_mapping.json

```diff
@@ -1,5 +1,27 @@
 {
-    "index_config": [{
+    "index_config": [
```

Comment:
Excellent choice on the object field type. I don't think you need the normalizer to lower there. That's needed in the other indexes for name keywords, but that doesn't apply here.